flowmachine/tests/test_displacement.py

```diff
@@ -127,7 +127,9 @@ def test_get_all_users_in_reference_location(get_dataframe):
     p2 = ("2016-01-01 12:01:00", "2016-01-01 15:20:00")
 
     rl = daily_location("2016-01-01", spatial_unit=make_spatial_unit("lon-lat"))
-    d = Displacement(p2[0], p2[1], reference_location=rl)
+    d = Displacement(
+        p2[0], p2[1], reference_location=rl, return_subscribers_not_seen=True
```

Comment:
Can we also have a test for the new behaviour with `return_subscribers_not_seen=False`?